apps/laboratory/package.json

```diff
@@ -5,7 +5,11 @@
   "scripts": {
     "dev:laboratory": "next dev",
     "build:laboratory": "next build",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+    "playwright:start": "next dev",
```

Comment:
If tests are for lab, this should be `npm run laboratory` to start in in dev mode. Optionally, you could combine with a fresh build as well, so `turbo run build; npm run laboratory`

Reply:
What's the difference between `npm run build` and `turbo run build`?